app/models/katello/host/content_facet.rb

```diff
@@ -15,12 +15,14 @@ class ContentFacet < Katello::Model
 
       has_many :content_facet_errata, :class_name => "Katello::ContentFacetErratum", :dependent => :delete_all, :inverse_of => :content_facet
       has_many :applicable_errata, :through => :content_facet_errata, :class_name => "Katello::Erratum", :source => :erratum
+      has_many :installable_errata, :through => :content_facet_errata, :class_name => "Katello::Erratum", :source => :erratum
```

Comment:
I don't think this makes sense, as the errata in this association are 'applicable', but not necessarily installable.  You'd need to call the installable_errata' method below to calculate that.  Same comment for the installable_rpms relationship down below